### PR TITLE
lint and cleanup code

### PR DESCRIPTION
**Pull request by saxix**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -74,6 +46,7 @@ def run_tests(self):
         'Programming Language :: Python',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.6',
```
Comment (domdinicola) [inline setup.py]: Shall we add Django also here?

Comment (saxix) [reply setup.py]: classifiers are not mandatory and only useful for "real" public project. we can opt to add Django but what about others? https://pypi.org/classifiers/

Comment (domdinicola) [reply setup.py]: at least we should have Development Status, Python, Django

**Comment by reinbach** [comment]
I thought we wanted to move away from `pipenv`?

**Pull request merged**